main.tf

```diff
@@ -108,25 +110,27 @@ docker-machine create --driver google \
     --google-machine-image ubuntu-os-cloud/global/images/ubuntu-2004-focal-v20220419 \
     --google-tags ${var.ci_worker_instance_tags} \
     --google-use-internal-ip \
+    --google-network ${var.ci_runner_network} \
+     %{if var.ci_runner_subnetwork != ""}--google-subnetwork ${var.ci_runner_subnetwork}%{endif} \
     ${var.gcp_resource_prefix}-test-machine
 
-docker-machine rm -y ${var.gcp_resource_prefix}-test-machine
+# This is sometimes hanging, machine is correctly deleted but the check seems to be failing
+# So putting that in background instead to not block the rest of the process
+docker-machine rm -y ${var.gcp_resource_prefix}-test-machine &
```

Comment:
How often was this happening? I've never seen it occur. Admittedly I don't deploy this very often.